components/Overlay/stories.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { Sample } from "../../lib/stories-common";
+
+import Overlay from "./index";
+
+storiesOf("Overlay", module)
+  .add("Everything", () => (
+    <Sample notes={"This is a simple Overlay, with a popup to illustrate the use of an overlay."}>
+      <Overlay/>
```

Comment:
Finally, after adding support for children in the React component, you can include the popup sample here like so:
```jsx
<Overlay>
  <div className="popup"></div>
</Overlay>
```